libsplinter/src/service/scabbard/client/mod.rs

```diff
@@ -0,0 +1,306 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod contract_registry;
+mod error;
+mod execute;
+mod key;
+mod namespace;
+mod smart_permission;
+mod submit;
+mod transaction;
+mod upload;
+
+use std::fs::File;
+use std::io::{BufReader, Read};
+
+use sabre_sdk::protocol::payload::SabrePayload;
+pub use sawtooth_sdk::signing::secp256k1::Secp256k1PrivateKey;
```

Comment:
This should not be re-exported as pub - import it directly where it is needed.

Reply:
Removed from this PR